src/app/pages/adoption-form/adoption-form.hbs

```diff
@@ -0,0 +1,136 @@
+<style scoped>@import url(/app/pages/adoption-form/adoption-form.css);</style>
+<h1>OpenStax College Adoption Form</h1>
+<p>
+    Future funding from the <a href="/about">generous foundation donors</a> who
+    support our work is based on the number of faculty who adopt and the impact
+    our resources have on students. Please complete the form below to help us
+    demonstrate how OpenStax College textbooks are making education more accessible!
+</p>
+<p>
+    Not using OpenStax College yet? Please complete the OpenStax
+    <a href="/adoption_interests">interest form</a> so we may send you the
+    information you need.
+</p>
+<form accept-charset="UTF-8" action="/adoptions" class="new_adoption"
+ id="new_adoption" method="post">
+    <div>
+        <input name="utf8" type="hidden" value="✓" />
+        <input name="authenticity_token" type="hidden" value="Xi14qRzCbSy2zkso1VEjKU4bEUfFWOrhWbQnh7tlO3c=" />
```

Comment:
This appears to be a CSRF token generated by rails, probably on the form you were duplicating.  It wouldn't have any meaning here, and would need to be randomly generated (and verified) by the server.

I'm all for using a CSRF token, but we need to have server support to do so.  So I think we either need to coordinate with @rich-hart and @mwharrison or just remove this line for the time being.